index.html

```diff
@@ -28,6 +28,7 @@ <h1>Directory of PWAs</h1>
     <li><a href="sample/index.html">Standalone PWA with random testing bits</a></li>
     <li><a href="title-vs-name/index.html">PWA with different title and name to showcase where each is used in the browser UI</a></li>
     <li><a href="displayWCO-notInstallable/index.html">PWA with "display": "window-controls-overlay" to verify that this display mode is ignored</a></li>
+    <li><a href="wco-empty/index.html">Empty WCO app. Because sometimes you need an empty app to test</a></li>
```

Comment:
nit: Can this be grouped with PWA display: [whatever] and read "PWA display_override: window-controls-overlay (Because sometimes you need an empty app to test)"? 

Keep the sass, but also group it with it's family 😉

Reply:
done.